### PR TITLE
Removing sugar annotation from samples.

### PR DESCRIPTION
**Pull request by n3wscott**
It is simpler to just make a Broker directly for samples.

### CODE REVIEW

File: docs/eventing/samples/helloworld/helloworld-go/README.md [hunk 1]
```diff
@@ -230,13 +235,14 @@ folder) you're ready to build and deploy the sample app.
...
-   1. Above command created a namespace `knative-samples` and labelled it with
-      `knative-eventing-injection=enabled`, to enable eventing in the namespace.
+   1. Above command created a namespace `knative-samples` and create a default Broker it.
       Verify using the following command:
 
       ```shell
-      kubectl get ns knative-samples --show-labels
+      kubectl get broker --namespace knative-samples
       ```
+
```
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-go/README.md:244]: Remove trailing whitespace:
```suggestion

```

File: docs/eventing/samples/helloworld/helloworld-go/README.md [hunk 2]
```diff
@@ -230,13 +235,14 @@ folder) you're ready to build and deploy the sample app.
...
-      `knative-eventing-injection=enabled`, to enable eventing in the namespace.
+   1. Above command created a namespace `knative-samples` and create a default Broker it.
       Verify using the following command:
 
       ```shell
-      kubectl get ns knative-samples --show-labels
+      kubectl get broker --namespace knative-samples
       ```
+      
+      _Note_: you can also use injection based on labels with the [Eventing Sugar Controller](../../../../install/any-kubernetes-cluster.md).
```
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-go/README.md:245]: Remove trailing whitespace:
```suggestion
      _Note_: you can also use injection based on labels with the [Eventing Sugar Controller](../../../../install/any-kubernetes-cluster.md).
```

File: docs/eventing/samples/helloworld/helloworld-go/README.md (hunk 1 above)
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-go/README.md:244]: Remove trailing whitespace:
```suggestion

```

File: docs/eventing/samples/helloworld/helloworld-go/README.md (hunk 2 above)
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-go/README.md:245]: Remove trailing whitespace:
```suggestion
      _Note_: you can also use injection based on labels with the [Eventing Sugar Controller](../../../../install/any-kubernetes-cluster.md).
```

File: docs/eventing/samples/helloworld/helloworld-go/README.md [hunk 3]
```diff
@@ -138,67 +138,73 @@ cd knative-docs/docs/eventing/samples/helloworld/helloworld-go
...
+   apiVersion: v1
+   kind: Namespace
+   metadata:
+     name: knative-samples
+   ---
+   # A default broker
+   apiVersion: eventing.knative.dev/v1
+   kind: Broker
+   metadata:
+     name: default
```
Comment (vaikas) [inline docs/eventing/samples/helloworld/helloworld-go/README.md:151]: Looks like there are mismatched namespaces? Above you use knative-samples?

File: docs/eventing/samples/helloworld/helloworld-go/sample-app.yaml [hunk 4]
```diff
@@ -1,10 +1,15 @@
...
 kind: Namespace
 metadata:
   name: knative-samples
-  labels:
-       knative-eventing-injection: enabled
+---
+# A default broker
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
```
Comment (vaikas) [inline docs/eventing/samples/helloworld/helloworld-go/sample-app.yaml:10]: same here?

**Pull request merged**